lib/string-utils.ts

```diff
@@ -70,11 +73,7 @@ export function convertToKebabCase(str: string): string {
  * @returns {string} kebab-cased version of `str`
  */
 export function getKebabCase(str?: string): string {
-  return convertToKebabCase(
-    removeSpaces(
-      replaceSpecialCharacterWithSpace(replaceSpacesWithCamelCase(str ?? "")),
-    ),
-  );
+  return convertToKebabCase(replaceSpecialCharacterWithSpace(str ?? "").trim());
```

Comment:
Unfortunately, the [Unit and integration tests CI check](https://github.com/kawalcovid19/wargabantuwarga.com/pull/313/checks?check_run_id=3151271445#step:4:111) still failed due to the lower code coverage.
Could you please write more tests to cover the rest of the functions in this `string-utils.ts` file?
Thanks, @fncolon! 🙏

Reply:
Oh i see, will write more tests, after take a sleep for few hours. 

Thanks.

Reply:
Thanks, have a nice rest, @fncolon, take care of yourself! 😄

Reply:
I just added some tests on 41ddae9 to increase the code coverage just enough to meet the threshold. 😁